tests/kernel/wave/attention/vanilla_attention_test.py

```diff
@@ -101,6 +101,7 @@ def testTransposedVAttentionPure(
         schedule=enable_scheduling,
         use_scheduling_barriers=enable_scheduling_barriers,
         dynamic_symbols=dynamic_symbols,
+        canonicalize=True,
```

Comment:
Do we need this? I thought Will landed a PR that makes canonicalize True by default.